ci_framework/roles/dlrn_report/tasks/dlrn_report_results.yml

```diff
@@ -0,0 +1,49 @@
+---
+- name: Set zuul-log-path fact
+  when: zuul is defined
+  ansible.builtin.include_role:
+    name: set-zuul-log-path-fact
+
+- name: Get hash related data from repo_setup role
+  ansible.builtin.include_role:
+    name: repo_setup
+
+- name: Report results to dlrn for the tested hash
+  ansible.builtin.shell:
+    cmd: >-
+      {% if zuul_success is defined and zuul_success |bool %}
+      echo "REPORTING SUCCESS TO DLRN API"
+      {% else %}
+      echo "REPORTING FAILURE TO DLRN API"
+      {% endif %}
+      export SSL_CA_BUNDLE="/etc/pki/ca-trust/extracted/openssl/ca-bundle.trust.crt"
+      dlrnapi --url {{ cifmw_repo_setup_dlrn_api_url }}
+      {% if cifmw_dlrn_report_dlrnapi_host_principal is defined and cifmw_dlrn_report_kerberos_auth|bool %}
+      --server-principal {{ cifmw_dlrn_report_dlrnapi_host_principal }}
+      --auth-method kerberosAuth
+      {% endif %}
+      report-result
+      {% if cifmw_repo_setup_full_hash is defined %}
+      --agg-hash {{ cifmw_repo_setup_full_hash }}
+      {% endif %}
+      {% if cifmw_repo_setup_extended_hash is defined %}
+      --extended_hash {{ cifmw_repo_setup_extended_hash }}
+      {% endif %}
+      {% if cifmw_repo_setup_commit_hash is defined %}
+      --commit_hash {{ cifmw_repo_setup_commit_hash }}
+      {% endif %}
+      {% if cifmw_repo_setup_distro_hash is defined %}
+      --distro_hash {{ cifmw_repo_setup_distro_hash }}
+      {% endif %}
+      --job-id {{ zuul.job }}
+      --info-url "{{ log_host_url | default('https://logserver.rdoproject.org') }}/{{ zuul_log_path }}"
+      --timestamp $(date +%s) \
+      --success {{ zuul_success | bool }}
+    chdir: '{{ cifmw_dlrn_report_workspace }}'
+  environment: |
+    {{ zuul | zuul_legacy_vars | combine({
+      'DLRNAPI_PASSWORD': dlrnapi.password,
```

Comment:
i would say that secret name could be a variable, but hardcoded to 'dlrnapi' will work for both rdo and DS

Reply:
Will do a followup patch after testing.